packages/connect-react/src/components/shared/Button.tsx

```diff
@@ -10,11 +10,29 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
   ({ isLoading = false, disabled, children, ...rest }, ref) => {
     return (
       <button
+        style={{ position: 'relative' }}
         disabled={isLoading || disabled}
         ref={ref}
         {...rest}
       >
-        {isLoading ? <LoadingSpinner variant='on-primary' /> : children}
+        {isLoading && (
+          <div
+            style={{
```

Comment:
so far we are not using inline styles at all (or at least not that much) => To keep it consistent I would avoid them and stick with the current way (defining a css class).